lib/src/Uncertainty/Model/DistributionImplementation.cxx

```diff
@@ -3832,10 +3852,17 @@ Graph DistributionImplementation::drawPDF(const Point & xMin,
   const GraphImplementation::LogScale scale = static_cast<GraphImplementation::LogScale>((logScaleX ? 1 : 0) + (logScaleY ? 2 : 0));
   if (isContinuous())
   {
-    Graph graph(PDFWrapper(this).draw(xMin, xMax, pointNumber, scale));
-    graph.setXTitle(description_[0]);
-    graph.setYTitle(description_[1]);
-    graph.setTitle(String(OSS() << getDescription() << " iso-PDF"));
+    // retrieve x/y discretization from a dummy graph
+    const Drawable dummyDrawable(LinearFunction(Point(2), Point(1), Matrix(1, 2, Collection<Scalar>({1.0, 1.0}))).draw(xMin, xMax, pointNumber, scale).getDrawable(0));
```

Comment:
Why not using `SymbolicFunction(["x", "y"],["x+y"])`? It is both faster (x6.64 on my laptop) and more readable. And if you go for `SymbolicFunction(["x", "y"],["0"])` you get a 10% boost!